website/server/controllers/api-v3/challenges.js

```diff
@@ -386,39 +386,52 @@ api.getUserChallenges = {
       query.categories = { $elemMatch: { slug: {$in: categorySlugs} } };
     }
 
-    let mongoQuery = Challenge.find(query)
-      .sort('-createdAt');
+    // Ensure that official challenges are always first
+    let mongoQuery = Challenge.aggregate()
```

Comment:
could you extract the Challenge-query to a function? that way if we need to fix this query we won't have to change two queries

Reply:
Done.